# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/interfaces/nsIAccessiblePivot.idl
# Commit: 786805a2cdcc
# Full Hash: 786805a2cdcc4cdab632a955dace417062d0087e
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r effe12f2120a -r 786805a2cdcc accessible/interfaces/nsIAccessiblePivot.idl
--- a/accessible/interfaces/nsIAccessiblePivot.idl	Tue Nov 28 22:03:20 2023 +0000
+++ b/accessible/interfaces/nsIAccessiblePivot.idl	Tue Nov 28 22:03:21 2023 +0000
@@ -20,13 +20,6 @@
 [scriptable, uuid(81fe5144-059b-42db-bd3a-f6ce3158d5e9)]
 interface nsIAccessiblePivot : nsISupports
 {
-  // XXX: These are here for the VC changed event that has yet to go away.
-  const PivotMoveReason REASON_NONE = 0;
-  const PivotMoveReason REASON_NEXT = 1;
-  const PivotMoveReason REASON_PREV = 2;
-  const PivotMoveReason REASON_FIRST = 3;
-  const PivotMoveReason REASON_LAST = 4;
-
   /**
    * Move pivot to next object, from current position or given anchor,
    * complying to given traversal rule.